shanoir-ng-front/src/app/dataset-acquisitions/tree/dataset-acquisition-node.component.ts

```diff
@@ -71,8 +71,9 @@ export class DatasetAcquisitionNodeComponent extends TreeNodeAbstractComponent<D
                     this.input.studyRights.includes(StudyUserRight.CAN_ADMINISTRATE),
                     this.input.studyRights.includes(StudyUserRight.CAN_DOWNLOAD)
                 );
-                this.loadDatasets();
             }
+            this.node.registerOpenPromise(this.contentLoaded);
```

Comment:
this is in order to make the tree work well with the fact that the acquisitions are no loaded in a separate time